src/tools/interop/idt/capture/ecosystem/play_services/analysis.py

```diff
@@ -0,0 +1,101 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import os
+
+from capture.file_utils import add_border, create_standard_log_name, print_and_write
+from capture.platform.android import Android
+
+
+class PlayServicesAnalysis:
```

Comment:
Not a problem for now, and you probably considered this already, but this does make the logs load-bearing for the interop lab, so we can't change the logs without breaking the capture tool.

I'm not sure I have a better way to do this, but if there are tags or something, it might be good to hook in that way.

Reply:
Yes this problem has consistently come up every time we discuss log parsing. Happy to discuss some possible solutions on the ecosystem side further out of band.

Right now, the patterns that are in this file are basically a tag and one more filter.